src/moderation.rs

```diff
@@ -109,23 +109,28 @@ pub fn joke_ban(args: &Args) -> Result<(), Error> {
         None => return crate::api::send_reply(args, "🤨"),
     };
 
-    let bannee = args.body.split_whitespace().next().and_then(|first_arg| {
+    let arguments: Vec<&str> = args.body.split(' ').collect();
+
+    let bannee = arguments.get(0).and_then(|first_arg| {
         parse_member(
             &guild_id.to_guild_cached(&args.cx.cache)?.read().members,
             first_arg,
         )
         .map(|m| m.user.read().clone())
     });
 
+    let reason = arguments.get(1..).map(|r| r.join(" "));
+
     match bannee {
         Some(bannee) => crate::api::send_reply(
             args,
             &format!(
-                "{}#{} banned user {}#{} {}",
+                "{}#{} banned user {}#{}{}  {}",
                 args.msg.author.name,
                 args.msg.author.discriminator,
                 bannee.name,
                 bannee.discriminator,
+                reason.unwrap_or("".to_string()),
```

Comment:
you can do `.unwrap_or_default()` too

Reply:
oh lol it has been merged

Reply:
Yes it's already merged heh

Sorry, I didn't know people were about to review the PR

Reply:
No need to be sorry 💛. I don't think a single comment qualifies as a "review" 😄.